packages/superset-ui-translation/src/types/index.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { Jed as BaseJed, JedOptions, DomainData, Translations } from './jed';
+
+export { Translations } from './jed';
+
+/**
+ * Superset supported languages.
+ */
+export type Locale = 'de' | 'en' | 'es' | 'fr' | 'it' | 'ja' | 'ko' | 'pt' | 'pt_BR' | 'ru' | 'zh'; // supported locales in Superset
```

Comment:
I can't think of an easy way to DRY this up (having it somehow synchronized with config.py), but that would be a nifty trick. Probably not worth worrying about, but it crossed my mind, anyway.

Reply:
I'd imagine it's fairly rare we need to update this so this is probably OK.